data/wv/legislature/Jim-Butler-b8c5f0c6-1e47-4fd9-9cc5-1550e2ba6848.yml

```diff
@@ -0,0 +1,26 @@
+id: ocd-person/b8c5f0c6-1e47-4fd9-9cc5-1550e2ba6848
+name: Jim Butler
+given_name: Jim
+family_name: Butler
+email: jim.butler@wvhouse.gov
+image: http://www.wvlegislature.gov/images/members/2023/house/butler_jim.jpg
+party:
+- name: Republican
+roles:
+- start_date: '2018-12-01'
+  type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:wv/government
+  district: '18'
+offices:
+- classification: capitol
+  address: Room 225E, Bldg. 1 1900 Kanawha Blvd. E. Charleston, WV 25305
+  voice: 304-340-3199
+- classification: district
```

Comment:
@sroomf this person should already have a file as they served in house district 14 from 12/01/2012 until 12/01/2020. We just need to add their new role:
- start_date: '2022-12-01'
  type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:wv/government
  district: '18'